app/views/layouts/admin_layout.html.erb

```diff
@@ -41,6 +41,22 @@
     </main>
   </div>
 
-  <%= render "govuk_publishing_components/components/layout_footer" %>
+  <%= render "govuk_publishing_components/components/layout_footer", {
+    navigation: [
+      {
+        title: "Support and feedback",
+        items: [
+          {
+            href: "https://support.publishing.service.gov.uk/",
```

Comment:
I think this should use `Plek` to find the support application: `Plek.new.external_url_for("support")`

Reply:
Updated and rebased